### DISCUSSION TITLE
Disable newly created Singles (if "homepage") to include duplicated content

### DISCUSSION THREAD

**Discussion by naboo** [craftcms/cms#12027] [Ideas]
I've always wondered why this is happening but the more and more use of multi-site installations of Craft it keeps getting annoying and it also makes the creation of new Singles (homepage) slower and slower as sites grow with new sites being added.

Do we need newly created Singles (if "homepage") to include duplicated content from the primary site homepage Single? If so, why?

If one creates a new site that site surely doesn't have the same content as another site.

**Comment by cmalven** [comment]
Running into the same issue. There is no benefit to duplicating existing content over to a new site homepage single on my current project, and it actually makes things more confusing for the content editors to have this content moved over.

**Comment by thupsi** [comment]
I haven't run into this issue, but I assume, if you have a multilang site and the homepage has complex matrix fields, it is helpfull to have the duplicated content as a starting point for your translation. I agree that the duplication should be optional though!
